src/components/InviteToOpenSauced/InviteToOpenSaucedButton.ts

```diff
@@ -0,0 +1,21 @@
+import logoIcon from "../../assets/opensauced-icon.svg";
+import "../../index.css";
+
+export const InviteToOpenSaucedButton = () => {
+  const inviteToOpenSaucedButton = document.createElement("a");
+  Object.assign(inviteToOpenSaucedButton, {
```

Comment:
We should probably keep this consistent across files.

Also, simply assigning object properties without using the Object prototype would be ok ( it's not needed).

I am fine with both approaches anyway, but consistently across component injections.

Reply:
I created a little utility for this a while back if you want to steal it. https://gist.github.com/nickytonline/8223b27b19c080c28d9f0d3b7fce1e82#file-createelement-js

Reply:
> I created a little utility for this a while back if you want to steal it. https://gist.github.com/nickytonline/8223b27b19c080c28d9f0d3b7fce1e82#file-createelement-js

This would make for a great utility function since there's a lot of element creation going around in the repo. Thanks.